submissions/Sergey-Kali/index.css

```diff
@@ -0,0 +1,190 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: lightgray;
+    overflow-x: hidden;
+}
+
+a {
+    text-decoration: none;
+}
+
+.nav-panel {
+    float: right;
+    display: flex;
+}
+
+.nav-list {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    list-style: none;
+    
+}
+
+.mail, 
+.img {
+    display: block;
+    padding: 12px 0px;
+    min-width: 80px;
+    text-align: center;
+}
+
+.bell img {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css) Avoid too specific CSS selectors. [Read about it](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)